### PR TITLE
Feat 444 organize warning labels

### PR DESCRIPTION
**Pull request by micahwoodard**
### Pull Request instructions:
- Please follow the [update protocol](https://github.com/AllenNeuralDynamics/aind-behavior-blog/wiki/Software-Update-Procedures)
- Answer the questions below in detail. Your responses will be emailed to experimenters. 
- If the experimenters must do anything new, provide detailed step by step instructions on the [wiki](https://github.com/AllenNeuralDynamics/aind-behavior-blog/wiki)
- If computer maintainers need to manually update anything, provide detailed step by step instructions
- Use markdown syntax in order for your comments to be rendered reliably in the email: "1." instead of "1)", use four spaces for indents.
- If you use the keyword "skip email" in the title, it will skip the email updates
- Merges from "develop" into "production_testing" should use the keyword "production merge" in the title for reliable indexing of updates
- Merges from "production_testing" into "main" should use the keyword "update main"
  
### Describe changes:
- warning box is now a widget what displays log warnings
### What issues or discussions does this update address?
- resolves https://github.com/AllenNeuralDynamics/dynamic-foraging-task/issues/444
### Describe the expected change in behavior from the perspective of the experimenter
-  Warning box will display newest log messages at top of box. Warning will not be removed anymore
### Describe any manual update steps for task computers
- None 
### Was this update tested in 446/447?
- [x] 446/447

### CODE REVIEW

**Comment by micahwoodard** [comment]
Added camera warnings back into dialog

File: src/foraging_gui/Dialogs.py [hunk 1]
```diff
@@ -1287,14 +1292,20 @@ def _connectSignalsSlots(self):
...
+
+        text = self.info_label.text()
         if hasattr(self.MainWindow,'Ot_log_folder'):
             try:
                 subprocess.Popen(['explorer', os.path.join(os.path.dirname(os.path.dirname(self.MainWindow.Ot_log_folder)),'behavior-videos')])
             except Exception as e:
                 logging.error(str(e))
                 logging.warning('No logging folder found!', extra={'tags': self.MainWindow.warning_log_tag})
+                if 'No logging folder found!' not in text:
+                    self.info_label.setText(text + '\n No logging folder found!')
```
Comment (micahwoodard) [inline src/foraging_gui/Dialogs.py:1304]: image

File: src/foraging_gui/Dialogs.py [hunk 2]
```diff
@@ -1313,6 +1324,8 @@ def _start_preview(self):
 
             self.StartPreview.setStyleSheet("background-color : green;")
             logging.info('Camera is on', extra={'tags': self.MainWindow.warning_log_tag})
+            self.info_label.setText('Camera is on')
```
Comment (micahwoodard) [inline src/foraging_gui/Dialogs.py:1327]: image

File: src/foraging_gui/Dialogs.py [hunk 3]
```diff
@@ -1323,6 +1336,7 @@ def _start_preview(self):
 
             self.StartPreview.setStyleSheet("background-color : none;")
             logging.info('Camera is off', extra={'tags': self.MainWindow.warning_log_tag})
+            self.info_label.setText('Camera is off')
```
Comment (micahwoodard) [inline src/foraging_gui/Dialogs.py:1339]: image

File: src/foraging_gui/Dialogs.py [hunk 4]
```diff
@@ -1338,6 +1352,7 @@ def _StartCamera(self):
         if self.StartRecording.isChecked():
             self.StartRecording.setStyleSheet("background-color : green;")
             logging.info('Camera is turning on', extra={'tags': self.MainWindow.warning_log_tag})
+            self.info_label.setText('Camera is turning on')
```
Comment (micahwoodard) [inline src/foraging_gui/Dialogs.py:1355]: image

File: src/foraging_gui/Dialogs.py [hunk 5]
```diff
@@ -1364,14 +1379,17 @@ def _StartCamera(self):
             time.sleep(5)
             self.camera_start_time = str(datetime.now())
             logging.info('Camera is on!', extra={'tags': [self.MainWindow.warning_log_tag]})
+            self.info_label.setText('Camera is on!')
         else:
             self.StartRecording.setStyleSheet("background-color : none")
             logging.info('Camera is turning off', extra={'tags': self.MainWindow.warning_log_tag})
+            self.info_label.setText('Camera is turning off')
```
Comment (micahwoodard) [inline src/foraging_gui/Dialogs.py:1386]: image

**Review by alexpiet — APPROVED** [review]

**Pull request merged**